scripts/calculate_ram_hours.py

```diff
@@ -70,13 +70,25 @@ def calculate_ram_hours(accession_code, start_date):
     for original_file_id in sample.original_files:
         original_file = pyrefinebio.OriginalFile.get(original_file_id)
 
+        seen_downloader_job_ids = set()
```

Comment:
You need to lift these sets outside the above `for` loop, because the overcounting happens across original files instead of within a single original file

Reply:
Oh good catch again, that was sloppy.